command/git.go

```diff
@@ -170,49 +203,27 @@ func (c *gitFetchProject) buildCloneCommand(conf *model.TaskConfig) ([]string, e
 	return gitCommands, nil
 }
 
-func (c *gitFetchProject) buildModuleCloneCommand(cloneURI, owner, repo, moduleBase, ref string, conf *model.TaskConfig, modulePatch *patch.ModulePatch) ([]string, error) {
-	if cloneURI == "" {
+// func (c *gitFetchProject) buildModuleCloneCommand(cloneURI, owner, repo, moduleBase, ref string, conf *model.TaskConfig, modulePatch *patch.ModulePatch, oauthToken string) ([]string, error) {
+func (c *gitFetchProject) buildModuleCloneCommand(conf *model.TaskConfig, opts cloneOpts, ref string, modulePatch *patch.ModulePatch) ([]string, error) {
+	gitCommands := []string{
+		"set -o xtrace",
```

Comment:
Does this potentially make us log the API token to the task?

Reply:
It shouldn't, because it does `set +o xtrace` right before it echoes the git clone command to unset xtrace.

This is tested in `TestTokenScrubbedFromLogger`.